account_invoice_supplierinfo_update/README.rst

```diff
@@ -0,0 +1,82 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================================================
+Update Supplier Info of product from Supplier Invoice
+=====================================================
+
+This module allows in the supplier invoice, to automatically update all
+products informations whose purchase informations on the line are different
+from the supplier informations, defined on the product form.
+
+It creates a new supplier information line if there is not or it updates the
+first.
+
+This module add an extra button 'Check Supplier Informations' on supplier
+invoice form.
+
+.. image:: /account_invoice_supplierinfo_update/static/description/supplier_invoice_form.png
+
+
+When user click on it, he can see supplier informations changes, that will
+be applied. Optionnaly, he can unselect some temporary changes, specially, if
+supplier applied an exceptionnal price change.
+
+.. image:: /account_invoice_supplierinfo_update/static/description/main_screenshot.png
+
+* blue: Create a full new supplier info
+* brown: Adding a partnerinfo line in a existing supplierinfo
+* black: Updating a current setting, displaying price variation (%)
+
+This module add an extra boolean field 'Supplier Informations Checked' in the
```

Comment:
This module adds an extra boolean field 'Supplier Informations Checked' in the
'Other Info' tab inside the supplier invoice form, that indicates that the prices
have been checked and supplierinfo updated (or eventually that the 
changes have been ignored).